src/System.Security.AccessControl/src/System.Security.AccessControl.csproj

```diff
@@ -120,6 +120,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.Collections" />
+    <Reference Include="System.Collections.NonGeneric" />
```

Comment:
Before we merge this -- @joperezr @weshaggard 

S.S.AccessControl builds for `netstandard` but S.C.NonGeneric builds for `netcoreapp` and `uap` specifically. So assume this reference is prohibited. But why does NonGenreic not build for `netstandard` only?